### DISCUSSION TITLE
`tar_visnetwork()` - Size of the frame?

### DISCUSSION THREAD

**Discussion by rkrug** [ropensci-books/targets#82] [Help]
Hi

I have a large targets pipeline and when I use `tar_visnetwork()`, it shows the pipeline as expected. 

But when I zoom in, the window (see screenshot below) which is filled by the pipeline is rather small compared to the available space on the screen. 

![Screenshot 2024-09-02 at 10 07 14](https://github.com/user-attachments/assets/90d1848a-7293-4653-919f-1cb1b1298990)

Is there a way of increasing the size of this window?

Thanks

Rainer

**Comment by wlandau** [comment]
You could try piping `tar_visnetwork()` into `visNetwork::visOptions()`:

```r
targets::tar_visnetwork() |>
  visNetwork::visOptions(width = "800px", height = "800px")
```

**Comment by rkrug** [comment]
Thanks - I will try that. Sorry for posting in the wrong repo.
